README.md

```diff
@@ -26,6 +26,10 @@ Also, with Steam Play/Proton, it is expected that you can run your games without
 
     flatpak install flathub com.valvesoftware.Steam.Utility.thcrap_steam_proton_wrapper
 
+**Note: For flatpak version of Steam, by default `/home/$USER` or `$HOME` directory is mapped to `/home/$USER/.var/app/com.valvesoftware.Steam`.
+A dialog box will ask whether to install thcrap to `/home/$USER/.local/share/thcrap`,
+but the real location is `/home/$USER/.var/app/com.valvesoftware.Steam/.local/share/thcrap`**
+
```

Comment:
I'd like to keep the installation guide straightforward. The user probably doesn't care where things end up being installed.

Reply:
I think the best solution would be using `$XDG_DATA_HOME`[1] instead of `/home/$USER/.local/share` for default location like this
```bash
[ -z "$XDG_DATA_HOME" ] && XDG_DATA_HOME="$HOME/.local/share"
THCRAP_FOLDER="$XDG_DATA_HOME/thcrap"
```

flatpak will overwrite[2] this env variable, so it will show correct path.
Then we don't need those extra notes, otherwise it will cause some confusion.

[1] https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html#variables
[2] https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html#f6

Reply:
Indeed, using `$XDG_DATA_HOME` shows the correct path inside the sandbox. It also seems more correct overall. Nice solution!